plugins/inputs/dpdk/README.md

```diff
@@ -253,27 +282,59 @@ dpdk,host=dpdk-host,dpdk_instance=l3fwd-power,command=/ethdev/stats,params=0 [fi
-dpdk,command=/ethdev/link_status,dpdk_instance=l3fwd-power,host=dpdk-host,params=0 status="UP",speed=10000,duplex="full-duplex" 1606310780000000000
-dpdk,command=/ethdev/link_status,dpdk_instance=l3fwd-power,host=dpdk-host,params=1 status="UP",speed=10000,duplex="full-duplex" 1606310780000000000
+dpdk,command=/ethdev/link_status,dpdk_instance=l3fwd-power,host=dpdk-host,params=0 status="UP",status_i=1,speed=10000,duplex="full-duplex" 1606310780000000000
+dpdk,command=/ethdev/link_status,dpdk_instance=l3fwd-power,host=dpdk-host,params=1 status="UP",status_i=1,speed=10000,duplex="full-duplex" 1606310780000000000
```

Comment:
Maybe name it `status_code` and also have explanations of what each value means?

Reply:
status = string value of the status
status_i = integer value of the status

Reply:
covered in the last commit

Reply:
Most other plugins use `status_code`, it would unify the plugins more..

Reply:
Thank you, will cover this in another commit

Reply:
@Hipska 
The term `status_code` is too closely associated with exit codes. 
Here, we are referring to the numerical interpretation of the "UP" or "DOWN" status. I would stick to `status_i`.

Reply:
In the code it seems to be referred as `linkStatus`, so something related to that might work. I really don't like the `_i`, it is too much confusing..

Reply:
I'm agree that 'status_code' is more related to http codes/exit code. I would like to avoid this naming.

I can rename  it "status_numeric" or you can propose other options.
What do you think?

Reply:
@Hipska 
I'm agree that `status_code `is more related to http codes/exit code. I would like to avoid this naming.

I can rename  it `status_numeric` or you can propose other options.
What do you think?

Reply:
Hi,  @Hipska.

Any proposals for different naming? 
Thank you.

Reply:
As mentioned before, I think anything relevant to its actual name in the code (`linkStatus`) would probably be fine..

Reply:
will rename to `link_status`. Thank you